core/src/main/java/org/apache/carbondata/core/scan/executor/util/QueryUtil.java

```diff
@@ -130,10 +129,9 @@
    * @param keyOrdinalList
    * @param generator
    * @return
-   * @throws KeyGenException
    */
-  public static byte[] getMaxKeyBasedOnOrinal(List<Integer> keyOrdinalList, KeyGenerator generator)
-      throws KeyGenException {
+  public static byte[] getMaxKeyBasedOnOrinal(
```

Comment:
Please optimize it to getMaxKeyBasedOnOrdinal

Reply:
Please check Orinal in the whole project.

Reply:
fixed